x509util/extensions.go

```diff
@@ -63,20 +64,21 @@ const (
 	IPType                  = "ip"
 	RegisteredIDType        = "registeredID"
 	PermanentIdentifierType = "permanentIdentifier"
+	HardwareModuleType      = "hardwareModule"
```

Comment:
Not a blocker, but maybe this should be `HardwareModuleNameType` and `hardwareModuleName` to be consistent

Reply:
Yes, it should be.

Reply:
Fixed with fd6d5e5